src/ape_ethereum/transactions.py

```diff
@@ -50,6 +51,9 @@ class AccessList(BaseModel):
 
 
 class BaseTransaction(TransactionAPI):
+    block_id: Optional[BlockID] = None
```

Comment:
Been thinking it might be good to have this in the base model for cross-referencing (except always filled out?)

Reply:
We need it for calls too, for checking historical values